docs/docs/external/getting-started/test-network/ledger-initialization.md

```diff
@@ -255,7 +255,39 @@ Initialize bond and token asset states and ownerships on the `network1` ledger b
+  * The user certificate files corresponding to the Corda networks also need to be copied into this folder.
+
+### Initializing the Corda Networks
 
+Once the Corda networks (`Corda_Network` and `Corda_Network2`) are launched, the client applications (built earlier) needs to be exercised to generate ledger state in both exporting/source and importing/destination networks in preparation to test asset transfer interoperation flows.
+
+#### Bootstrapping Networks and Application States
+The Corda network ledger (or _vault_ on each node) must be initialized with access control policies, verification policies, and security group information for the other networks (two Fabric networks and other Corda network).
+
+Bootstrap the Corda networks and application states as follows (_the following instructions will initialize either or both Corda networks, depending on which of those are up and running_):
+- Navigate to the `samples/corda/corda-simple-application` folder.
+- Run the following:
+  * On the node owned by `PartyA` in `Corda_Network`:
+    ```bash
+    CORDA_PORT=10006 ./clients/build/install/clients/bin/clients configure network
+    CORDA_PORT=10006 ./clients/build/install/clients/bin/clients transfer save-cert
+    ```
+  * On the node owned by `PartyB` in `Corda_Network2`:
+    ```bash
+    NETWORK_NAME=Corda_Network2 CORDA_PORT=30009 ./clients/build/install/clients/bin/clients configure network
+    NETWORK_NAME=Corda_Network2 CORDA_PORT=30009 ./clients/build/install/clients/bin/clients util save-cert
+    ```
+  The `save-cert` command above fetches the certificate of the party in base64 format and stores in the folder `clients/src/main/resources/config/credentials/remoteNetworkUsers` with filename `Corda_Network_UsersAndCerts.json` and `Corda_Network2_UsersAndCerts.json` corresponding to `Corda_Network` and `Corda_Network2` respectively. These files are referred during pledge to fetch the certificate of the recipient of the transferred asset, and during claim to fetch the certificate of the pledger of the asset. *This is because, asset transfer doesn't require both the parties being members of neither in the export nor in the import network. It's just sufficient that pledger is part of the asset export network and claimer is part of the asset import network*.
```

Comment:
Same comment as above. The users don't need to know how the scripts are manipulating certificates, etc. The instructions in these docs should be limited to telling the user how to achieve something in a sequence of clearly spelled-out steps.

Reply:
Done Rama, and moved this to the README file.